pandas/core/indexes/datetimelike.py

```diff
@@ -479,6 +481,7 @@ def max(self, axis=None, *args, **kwargs):
         numpy.ndarray.max
         """
         nv.validate_max(args, kwargs)
+        nv.validate_minmax_axis(axis)
```

Comment:
these need to be inside these functions

the is so smelly to have to call 2 functions here
pls fix